### PR TITLE
Implemented basic required tool functionality

### PR DESCRIPTION
**Pull request by jwlebert**
## Important Additions
- Created a Draggable.cs script for objects that need to be used on plots.
- Created some basic ScriptableObjects for hoes and watering cans

### CODE REVIEW

File: mfgjs2023/Assets/Scripts/Draggable.cs [hunk 1]
```diff
@@ -0,0 +1,79 @@
...
+public class Draggable : MonoBehaviour, IPointerDownHandler, IPointerUpHandler
+{
+    public GameObject draggedObject;
+    public GameObject clone;
+
+    public UseableObject tool;
+
+    private const int UILayer = 5;
+
+    void Start()
```
Comment (cmdbris) [inline mfgjs2023/Assets/Scripts/Draggable.cs]: Put private for readability

File: mfgjs2023/Assets/Scripts/Draggable.cs [hunk 2]
```diff
@@ -0,0 +1,79 @@
...
+    {
+        AddPhysicsRaycaster();
+    }
+
+    void Update()
```
Comment (cmdbris) [inline mfgjs2023/Assets/Scripts/Draggable.cs]: Put private for readability

**Review by cmdbris — COMMENTED** [review]
Good, but explicitly put private for readability sake

File: mfgjs2023/Assets/Scripts/Draggable.cs (hunk 1 above)
Comment (jwlebert) [reply mfgjs2023/Assets/Scripts/Draggable.cs]:
```suggestion
    private void Start()
```

File: mfgjs2023/Assets/Scripts/Draggable.cs (hunk 2 above)
Comment (jwlebert) [reply mfgjs2023/Assets/Scripts/Draggable.cs]:
```suggestion
    private void Update()
```

**Review by cmdbris — APPROVED** [review]
AAAAHHHHH

**Pull request merged**